reporting_app/templates/plate_view.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'untabbed_datatables.html' %}
```

Comment:
This file is renamed from `library.html` with some refactoring to enable it to be used for genotyping